index.js

```diff
@@ -127,15 +127,19 @@ class Schematic {
    * @param {Vec3} newBlockState mc ver 1.11+
    * @returns {Array<string>} array of commands
    */
-  async makeWithCommands (offset) {
+  async makeWithCommands (offset, platform = 'pc') {
     const cmds = []
     await this.forEach(async (block, pos) => {
       const { x, y, z } = pos.offset(offset.x, offset.y, offset.z)
       const versionedMcData = mcData(this.version)
       let state
       if (versionedMcData.isNewerOrEqualTo('1.13')) {
         state = Object.entries(block.getProperties()).map(([key, value]) => `${key}="${value}"`).join(',')
-        state = state ? `[${state}]` : ''
+        if (platform === 'pc') {
+          state = state ? `[${state}]` : ''
+        } else {
+          state = state ? ` [${state}]` : ''
```

Comment:
Nit: I'd suggest explicitly checking if the platform is `'pe'` here and throw an error otherwise.

Reply:
Fixed